src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/IconTintColor/IconTintColorEffectRouter.android.cs

```diff
@@ -50,17 +51,21 @@ void ApplyTintColor()
 
 		void ClearTintColor()
 		{
-			switch (Control)
+			try
 			{
-				case ImageView image:
-					image.ClearColorFilter();
-					break;
-				case Button button:
-					var drawables = button.GetCompoundDrawables().Where(d => d != null);
-					foreach (var img in drawables)
-						img.ClearColorFilter();
-					break;
+				switch (Control)
+				{
+					case ImageView image:
+						image.ClearColorFilter();
+						break;
+					case Button button:
+						var drawables = button.GetCompoundDrawables().Where(d => d != null);
+						foreach (var img in drawables)
+							img.ClearColorFilter();
+						break;
+				}
 			}
+			catch (ObjectDisposedException ex) { }
```

Comment:
```suggestion
			catch (ObjectDisposedException ex) 
			{
				// We ignore ObjectDisposedException as a workaround of the XF issue
			}
```

Reply:
@AndreiMisiukevich Good "catch" :) I forgot that when I was debugging 👍🏻